src/tools/opt-dist/src/environment.rs

```diff
@@ -0,0 +1,100 @@
+use camino::Utf8PathBuf;
+use derive_builder::Builder;
+
+#[derive(Builder)]
```

Comment:
if you're deriving builders, is it worth deriving getters also?

Reply:
I don't think that `derive_builder` allows that. I think that the current code is fine, there's just a few functions, we could even make the fields public I suppose.